### ISSUE TITLE
feat(mat-option): Add aria-selected="false" back when used with single-selection

### ISSUE THREAD

**Issue by clamli** [angular/components#26642]
### Feature Description

When mat-option is used with single-selection, the unselected option does not have aira-selected="false" set. This violates WCAG Success Criteria: 4.1.2 Name, Role, Value (A) as [the aria-selected attribute is a required attribute for elements with role="option"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected).

### Expected behavior
- When mat-option is not selected, set aria-selected="false"
- When mat-option is selected, set aria-selected="true"

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
